docs/backuprestore-with-oadp.md

```diff
@@ -431,7 +437,99 @@ spec:
 > 1. The examples provided are just for reference. Create your own backup and restore CRs based on your needs.
 > 2. The backup and restore CRs must be created in the same namespace where the OADP is installed which is `openshift-adp`.
 > 3. The OADP configmaps should only contain backup and restore CRs which are in YAML format or multiple document YAML format. Other type of CRs are unknown to the LCA and will be ignored.
->
+
+##### 1.1 Backup resource when LVMS is used
+
+When LVMS is used to provide dynamic storage in the cluster, the backup and restore resources for the
+app should also include some extra fields to ensure each PV content is maintained when performing an IBU.
+
+backup_app.yaml
+
+```yaml
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: small-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  includedClusterScopedResources:
```

Comment:
Nit: it may be worth noting that this is only necessary exactly once per cluster restore since it is cluster scoped. (although specifying this per application is redundant, it also wont break anything)

Reply:
k, will drop a quick side note for that as well